exercises/practice/bob/.approaches/config.json

```diff
@@ -0,0 +1,35 @@
+{
+  "introduction": {
+    "authors": ["bobahop", "erikschierboom"]
+  },
+  "approaches": [
+    {
+      "uuid": "c0bab2cf-3304-480c-a454-f8dfd274883e",
+      "slug": "if",
+      "title": "If",
+      "blurb": "Use if and else to return the answer.",
```

Comment:
The sample code doesn't use `else`, only `if`. Does the approach still cover elses?

Reply:
Thanks! I need to correct the blurb.